.github/CODEOWNERS

```diff
@@ -1,12 +1,61 @@
 # Codeowners for IREE Github Repository.
 # Refer to https://docs.github.com/en/repositories/managing-your-repositorys-settings-and-features/customizing-your-repository/about-code-owners
-# for syntax of this file.
+# for syntax of this file (tl;dr: syntax is like .gitignore. Last matching rule
+# takes precedence).
+# Because of the precedence, rules for directories are listed topologically.
 
-# Global code owners
+# Default code owners to repo admins
+* @benvanik @GMNGeoffrey @ScottTodd @stellaraccident
```

Comment:
Yeah... It would be nice if there was something more like a round robin or "suggest reviewers". Looks like there are some options on GitHub, though none of them quite look like what I want: https://docs.github.com/en/organizations/organizing-members-into-teams/managing-code-review-settings-for-your-team#about-auto-assignment. Maybe we could create a team for GitHub global owners and then do one of the assignment algorithms. Seems like more work than I want to put into it right now though. Should I just drop the global owners?

Reply:
I could also just put myself as sacrificial lamb. I already get assigned all issues by default :-P

Reply:
I think just manually assigning reviewers works as a good default, so yes - drop the global owners. This CODEOWNERS mechanism looks most useful for letting some reviewers watch certain directories and to ensure that changes to those directories get sent to those maintainers for input.

Reply:
Yeah. I think I was probably still thinking in an "OWNERS approval required" mindset